.github/workflows/backend-integration-test.yml

```diff
@@ -80,6 +80,10 @@ jobs:
         if: ${{ contains(github.event.pull_request.labels.*.name, 'debug action') }}
         uses: csexton/debugger-action@master
 
+      - name: Free up disk space
```

Comment:
There is also a front-end integration test CI. Can you also add it as the CI will start Gravitino as well.

Reply:
OK. I will try.  I wonder that deleting some files might affect the front-end E2E tests. I'll do some tests later

Reply:
I add it to front-end integration test CI. It works well.  I had add 'Free up disk space' to front-end

https://github.com/datastrato/gravitino/actions/runs/8446045264/job/23134221228?pr=2597